src/tools/create-http-patch.js

```diff
@@ -18,10 +18,20 @@ const createHttpPatch = event => {
 
     // Proxy the request method
     object.request = (options, callback) => {
-      const requestUrl =
-        options.url ||
```

Comment:
I believe this isn't re-added anywhere, right? So this will break existing code that is passing in objects with a `.url` property, which may not be intended.

Reply:
If I understand the code correctly, `object.request` is to patch Node's [`http.request`](https://nodejs.org/docs/latest-v6.x/api/http.html#http_http_request_options_callback) method, which does not accept an object with a `.url` property:

```
> let http = require('http');
undefined
> http.request({url:'http://httpbin.org/get'});
...
> Error: connect ECONNREFUSED 127.0.0.1:80
    at Object.exports._errnoException (util.js:1018:11)
    at exports._exceptionWithHostPort (util.js:1041:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1090:14)
```

 If the point is to follow http.request's spec, then we should not accept an object with a `.url` property here neither. But I'm not sure if any existing code or library is doing that (passing an object with a `.url` property). Any clues about that?

Reply:
yeah, fair deal! should be safe to go then.